assets/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
I recommend putting index.html in your root of your repository because the url to your website on Github pages will then be: https://shriji96.github.io/Basic-Portfolio INSTEAD OF https://shriji96.github.io/Basic-Portfolio/assets/index.html